### PR TITLE
installer/jobs-reader role: limit rules

### PR DESCRIPTION
**Pull request by iMax-pp**

### CODE REVIEW

File: charts/pega/charts/installer/templates/pega-installer-role.yaml [hunk 1]
```diff
@@ -5,7 +5,13 @@ metadata:
...
-- apiGroups: ["", "batch", "extensions", "apps"]
-  resources: ["jobs", "deployments", "statefulsets"]
+- apiGroups: ["apps"]
+  resources: ["deployments", "statefulsets"]
+  verbs: ["get", "watch", "list"]
+- apiGroups: ["batch"]
+  resources: ["jobs"]
+  verbs: ["get", "watch", "list"]
+- apiGroups: ["extensions"]
+  resources: ["deployments"]
```
Comment (MadhuriArugula) [inline charts/pega/charts/installer/templates/pega-installer-role.yaml]: @iMax-pp Is there any specific reason we are splitting the api groups and verbs?

File: charts/pega/charts/installer/templates/pega-installer-role.yaml [hunk 2]
```diff
@@ -5,7 +5,13 @@ metadata:
   name: {{ template "installerJobReaderRole" }}
   namespace: {{ .Release.Namespace }}
 rules:
-- apiGroups: ["", "batch", "extensions", "apps"]
```
Comment (MadhuriArugula) [inline charts/pega/charts/installer/templates/pega-installer-role.yaml]: I didn't the reason for removing core API group.

File: charts/pega/charts/installer/templates/pega-installer-role.yaml (hunk 1 above)
Comment (iMax-pp) [reply charts/pega/charts/installer/templates/pega-installer-role.yaml]: I splitted api groups and resources to allow for different verbs on each couple of api group / resource.

File: charts/pega/charts/installer/templates/pega-installer-role.yaml (hunk 2 above)
Comment (iMax-pp) [reply charts/pega/charts/installer/templates/pega-installer-role.yaml]: I restricted the rules, as it was not working on the cluster I'm deploying on. But with these restricted rights, the installer deployment is working fine.

**Pull request merged**